src/emulator/gui/src/gui.cpp

```diff
@@ -278,7 +281,14 @@ void draw_ui(HostState &host) {
 
     if (host.gui.configuration_menu.settings_dialog)
         draw_settings_dialog(host);
-
+#ifdef NDEBUG
```

Comment:
unrelated to gui, discard changes in this file and find somewhere else to put this like app.cpp

Reply:
About the NDEBUG stuff. It's necessary but its kinda confusing. I kinda wish there was another definition that would enable discord (like USE_DISCORD added release). It would make the intention clearer- but its not a big deal. Just throwing stuff around lol.

Reply:
Yea, @IllusionMan1212 try making it a CMake option like [gdb stuff](https://github.com/Vita3K/Vita3K/blob/master/CMakeLists.txt#L8) and you'd need to do [this](to use it as a define in C++).
You can then use the `CI` CMake var (see line 36 there for example) to enable it on CI builds.